api-spec/STAC.yaml

```diff
@@ -928,13 +934,14 @@ components:
     itemCollectionLinks:
       type: array
       description: >-
-        An array of links to resources related to this collection.
+        An array of links. Can be used for pagination, e.g. by providing a link
+        with the `next` relation type.
       items:
         $ref: '#/components/schemas/link'
       example:
         - rel: next
           href: >-
-            http://api.cool-sat.com/stac/search?next=ANsXtp9mrqN0yrKWhf-y2PUpHRLQb1GT-mtxNcXou8TwkXhi1Jbk
+            http://api.cool-sat.com/query/gasd312fsaeg/ANsXtp9mrqN0yrKWhf-y2PUpHRLQb1GT-mtxNcXou8TwkXhi1Jbk
```

Comment:
?next is correct, but I incorrectly modified this file instead of the source one. I have another outstanding PR that fixes that.

Reply:
Which one is it? I could cherry pick from there...

Reply:
it's probably just easiest if you replace `http://api.cool-sat.com/query/gasd312fsaeg/ANsXtp9mrqN0yrKWhf-y2PUpHRLQb1GT-mtxNcXou8TwkXhi1Jbk` with `http://api.cool-sat.com/stac/search?next=ANsXtp9mrqN0yrKWhf-y2PUpHRLQb1GT-mtxNcXou8TwkXhi1Jbk` everywhere it exists

Reply:
Fixed it, thanks.